### PR TITLE
API Composer and Components docs

### PR DESCRIPTION
**Pull request by kimsauce**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/api-testing/composer.md [hunk 1]
```diff
@@ -1,77 +1,91 @@
...
+This button displays all available [assertion components](/api-testing/composer/assertion-components/), [I/O components](/api-testing/composer/io-components/), and [logical components](/api-testing/composer/logical-components/).<br/>
+<img src={useBaseUrl('img/api-fortress/2021/01/addComponent.png')} alt="Add Component"/>
 
-  <img src={useBaseUrl('img/api-fortress/2021/01/codeView.png')} alt="Test Composer Code View Pic"/>
+If a component is not valid for the operation you are conducting, it will not be made available to help avoid mistakes. For instance, if you don’t add a `POST` first, you cannot add a `POST` Body or `POST` Param.
 
-:::tip
-For information on API Testing Best Practices, visit our [Sauce School API Testing course](https://training.saucelabs.com/apiTesting/index.html).
+:::note
+Sauce Labs free trials do not give you access to all available components.
```
Comment (LoryApiFortress) [inline docs/api-testing/composer.md:35]: Sauce Labs free trials may do not give you access to all available components.

File: docs/api-testing/composer.md [hunk 2]
```diff
@@ -1,77 +1,91 @@
...
+<img src={useBaseUrl('img/api-fortress/2021/01/runTest.png')} alt="Run Test"/>
 
-6. __Save Test__: This button saves your progress.
+## Input Sets
+Displays the Input Set view where you can store input data sets to reuse across all your tests.<br/>
+<img src={useBaseUrl('img/api-fortress/2021/01/inputSets.png')} alt="Input Sets" width="500"/>
 
-7. __Run Test__: This button executes a test.
+There are two types of input data sets you can use:
+* __Global Parameters__: variables that are available across all tests in the project. Reference these variables simply by calling it within the test using the convention `“${VARIABLE}”`.
```
Comment (LoryApiFortress) [inline docs/api-testing/composer.md:68]: Global Parameters are variables that are available to be used throughout a test.

File: docs/api-testing/composer/logical-components.md [hunk 3]
```diff
@@ -0,0 +1,393 @@
...
+  </tbody>
+</table>
+
+<img src={useBaseUrl('img/api-fortress/2020/12/while.jpg')} alt="while.jpg"/>
+
+</details>
+
+
+
+## Other Components
```
Comment (LoryApiFortress) [inline docs/api-testing/composer/logical-components.md:114]: Are all these components inside the Logical Components section? They shouldn't

**Review by kimsauce — COMMENTED** [review]
@LoryApiFortress are you able to see the comments here? sorry for the confusion!

**Comment by LoryApiFortress** [comment]
> @LoryApiFortress are you able to see the comments here? sorry for the confusion!

yes

**Pull request merged**